InternetHospital.BusinessLogic/Services/DoctorService.cs

```diff
@@ -140,6 +144,29 @@ public DoctorProfileModel GetDoctorProfile(int userId)
             return doctorModel;
         }
 
+        public async Task<IActionResult> UpdateDoctorAvatar(string doctorId, IFormFile file)
```

Comment:
Dont know if its okay to return IActionResult inside serivce

Reply:
changed return type